app/views/time_based_price_policies/_amount_row.html.haml

```diff
@@ -19,9 +19,15 @@
         = price_policy.class.human_attribute_name(:full_price_cancellation)
         = tooltip_icon "fa fa-question-circle-o", t("price_policies.charge_full_price_on_cancellation_hint")
 
-%td
-  = pp.label :usage_rate, t("price_policies.rate"), class: "normal-weight"
-  = pp.text_field :usage_rate, value: display_usage_rate(price_group, price_policy),
-    size: 8,
-    class: "#{price_group.master_internal? ? 'master_usage_cost' : ''} usage_rate",
-    data: { target: ".js--usageRate" }
+- if @product.daily_booking?
+  %td.daily_rate
+    = pp.label :usage_rate_daily, t("price_policies.rate"), class: "normal-weight"
+    = pp.text_field :usage_rate_daily, value: display_rate(price_policy.usage_rate_daily),
```

Comment:
The `display_usage_rate` function returns a value from params if present and it is needed due to this bugfix https://github.com/tablexi/nucore-open/pull/340 which shouldn't happen for daily rates since there's no conversion from hours to minutes.

Btw, I'm not even sure if the `display_usage_rate` logic is needed at all since I couldn't reproduce the bug when removing the fix, but that's a conversation for another day.